js/admin.js

```diff
@@ -110,6 +110,7 @@ jQuery( document ).ready(function($) {
 							}
 						}
 					};
+					$('.progress-bar').addClass('install-error');
```

Comment:
I think this looks strange but we could ask @ernilambar 
![image](https://cloud.githubusercontent.com/assets/1785641/26530791/0aef0566-43dc-11e7-9ea7-3108f30dcaaa.png)

Perhaps we could adapt the text to mention that a check failed.

This looks great.
![image](https://cloud.githubusercontent.com/assets/1785641/26530802/67b582b6-43dc-11e7-851a-32485b79fd5b.png)